src/languageservice/services/yamlCompletion.ts

```diff
@@ -663,7 +663,7 @@ export class YamlCompletion {
         //    - item1
         // it will treated as a property key since `:` has been appended
         if (nodeParent && isSeq(nodeParent) && schema.schema.type !== 'object') {
-          this.addSchemaValueCompletions(schema.schema, separatorAfter, collector, {});
+          this.addSchemaValueCompletions(schema.schema, separatorAfter, collector, {}, isArray);
```

Comment:
If this is the only place `isArray` is used can we delay its initialization to inside this `if` block?